doc/man1/openssl-spkac.pod.in

```diff
@@ -93,6 +94,11 @@ SPKAC is being created).
 Output the public key of an SPKAC (not used if an SPKAC is
 being created).
 
+=item B<-md>
+
+The message digest used for signing. Historically the default value has
+been MD5, but SHA256 is recommended.
+
```

Comment:
Shouldn't it be specified by default in code too?

Reply:
Happy to specify sha256 by default, openssl happy with the change in default behaviour?